mysqli/mysqli.php

```diff
@@ -2109,6 +2125,7 @@ function mysqli_kill(mysqli $mysql, int $process_id): bool {}
  * @link https://php.net/manual/en/mysqli.set-local-infile-default.php
  * @param mysqli $mysql A link identifier returned by mysqli_connect() or mysqli_init()
  * @return void
+ * @removed 5.5
```

Comment:
for the reference: https://github.com/php/php-src/commit/522595086b8d654a2fd954977f7f443f2578de22